web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/entity-table-ui.component.html

```diff
@@ -164,7 +164,7 @@
       *ngFor="let srcProp of uniqueSourceFields(this.srcProps)"
       [ngStyle]="IndentCondition(srcProp)"
       (click)="insertField(srcProp, indx, prop) "
-    >{{displaySourceField(srcProp)}}</div>
+    ><span id="namespace">{{displaySourceField(srcProp).indexOf(':') != -1 ? '('+displaySourceField(srcProp).slice(0,displaySourceField(srcProp).indexOf(':'))+')' : ''}}</span><span id="field-no-namespace">{{displaySourceField(srcProp).indexOf(':') != -1 ? displaySourceField(srcProp).slice(displaySourceField(srcProp).indexOf(':')) : displaySourceField(srcProp)}}</span></div>
```

Comment:
Can the formatting here be cleaned up some for readability? The following doesn't seem to affect rendering of the paths in the menu:
```
     <div
      mat-menu-item
      class="mat-menu-list-btn"
      *ngFor="let srcProp of uniqueSourceFields(this.srcProps)"
      [ngStyle]="IndentCondition(srcProp)"
      (click)="insertField(srcProp, indx, prop) "
    >
      <span id="namespace">{{displaySourceField(srcProp).indexOf(':') != -1 ? '('+displaySourceField(srcProp).slice(0,displaySourceField(srcProp).indexOf(':'))+')' : ''}}</span>
      <span id="field-no-namespace">{{displaySourceField(srcProp).indexOf(':') != -1 ? displaySourceField(srcProp).slice(displaySourceField(srcProp).indexOf(':')) : displaySourceField(srcProp)}}</span>
    </div>
```

Reply:
@wooldridge - Made this change. Please review again.